docs/ADD_NEW_EXCHANGE.md

```diff
@@ -61,37 +61,36 @@ config.GetDefaultFilePath()
    "websocketOrderbookBufferLimit": 5,
    "baseCurrencies": "USD",
    "currencyPairs": {
-    "assetTypes": [
-      "spot",
-      "futures"
-     ],
-     "pairs": {
-      "futures": {
-       "enabled": "BTC-PERP",
-       "available": "BTC-PERP",
-       "requestFormat": {
-        "uppercase": true,
-        "delimiter": "-"
-       },
-       "configFormat": {
-        "uppercase": true,
-        "delimiter": "-"
-       }
+    "lastUpdated": 0,
+    "pairs": {
+     "futures": {
+      "assetEnabled": true,
+      "enabled": "SHIT-PERP",
+      "available": "SHIT-PERP",
+      "requestFormat": {
+       "uppercase": true,
+       "delimiter": "-"
+      },
+      "configFormat": {
+       "uppercase": true,
+       "delimiter": "-"
+      }
+     },
+     "spot": {
+      "assetEnabled": true,
+      "enabled": "BULLSHIT/USD",
```

Comment:
Can this be reverted back to  BTC/USD

Has a higher chance of not being delisted by the exchange later down the track and to not potentially offend anyone in future